### PR TITLE
Fix minor issues reported by static analysis

### PR DESCRIPTION
**Pull request by jantonguirao**
Signed-off-by: Joaquin Anton <janton@nvidia.com>

#### Why we need this PR?
*Pick one, remove the rest*
- It fixes minor issues reported by static analysis.

#### What happened in this PR?
*Fill relevant points, put NA otherwise. Replace anything inside []*
 - What solution was applied:
     *N/A*
 - Affected modules and functionalities:
     *N/A*
 - Key points relevant for the review:
     *N/A*
 - Validation and testing:
     *N/A*
 - Documentation (including examples):
     *N/A*

**JIRA TASK**: *[DALI-1587]*

### CODE REVIEW

File: dali/benchmark/normal_distribution_gpu_bench.cc [hunk 1]
```diff
@@ -42,6 +42,7 @@ BENCHMARK_DEFINE_F(OperatorBench, NormalDistributionGPU_NonUniform)(benchmark::S
   TensorListShape<3> shape(batch_size);
   int vol_log2 = 0;
   for (int i = 1; i < max_sample_vol; i *= 2) vol_log2++;
+  assert(vol_log2 > 0);
```
Comment (JanuszL) [inline dali/benchmark/normal_distribution_gpu_bench.cc:45]: I don't think it is recognized by the tool as a value check as it happens only in the debug build.

Comment (jantonguirao) [reply dali/benchmark/normal_distribution_gpu_bench.cc:45]: I know, but it serves as a debug check and as documentation saying that this value must be bigger than zero. We just need to mark it as intentional in Coverity

**Review by JanuszL — APPROVED** [review]

**Review by szalpal — APPROVED** [review]

File: dali/operators/coord/coord_flip.h [hunk 2]
```diff
@@ -95,10 +95,9 @@ class CoordFlip : public Operator<Backend> {
   }
 
 
-
   TensorLayout layout_;
-  int ndim_;
-  int x_dim_, y_dim_, z_dim_;
+  int ndim_ = -1;
+  int x_dim_ = -1, y_dim_ = -1, z_dim_ = -1;
```
[the remark this answers is not in the document]
Comment (jantonguirao) [reply dali/operators/coord/coord_flip.h:100]: will do

**Comment by jantonguirao** [comment]
!build

**Pull request merged**